src/videowidget.cpp

```diff
@@ -391,6 +400,12 @@ int VideoWidget::reconfigure(bool isMulti)
             m_consumer->set("color_trc", "bt709");
             break;
         }
+        if (property("processing_mode").toInt() == ShotcutSettings::Linear8Cpu
+            || property("processing_mode").toInt() == ShotcutSettings::Linear10Cpu) {
```

Comment:
CPU is not converting linear to HLG; so, here is what I suggest (`processingMode` is defined above):
```suggestion
        if (processingMode == ShotcutSettings::Linear8Cpu
            || processingMode == ShotcutSettings::Linear10Cpu
            || (processingMode == ShotcutSettings::Native10GpuCpu
                && property("decklinkGamma").toInt() != 1)) {
```